homeassistant/components/buienradar/sensor.py

```diff
@@ -58,132 +59,272 @@
+    "windforce": ["Wind force", "Bft", "mdi:weather-windy", None],
+    "winddirection": ["Wind direction", None, "mdi:compass-outline", None],
+    "windazimuth": ["Wind direction azimuth", DEGREE, "mdi:compass-outline", None],
+    "pressure": ["Pressure", PRESSURE_HPA, "mdi:gauge", None],
+    "visibility": ["Visibility", LENGTH_KILOMETERS, None, None],
+    "windgust": ["Wind gust", SPEED_KILOMETERS_PER_HOUR, "mdi:weather-windy", None],
     "precipitation": [
         "Precipitation",
         PRECIPITATION_MILLIMETERS_PER_HOUR,
         "mdi:weather-pouring",
+        None,
+    ],
+    "irradiance": [
+        "Irradiance",
+        IRRADIATION_WATTS_PER_SQUARE_METER,
+        "mdi:sunglasses",
+        None,
     ],
-    "irradiance": ["Irradiance", IRRADIATION_WATTS_PER_SQUARE_METER, "mdi:sunglasses"],
     "precipitation_forecast_average": [
         "Precipitation forecast average",
         PRECIPITATION_MILLIMETERS_PER_HOUR,
         "mdi:weather-pouring",
+        None,
     ],
     "precipitation_forecast_total": [
         "Precipitation forecast total",
         LENGTH_MILLIMETERS,
         "mdi:weather-pouring",
+        None,
     ],
     # new in json api (>1.0.0):
-    "rainlast24hour": ["Rain last 24h", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
+    "rainlast24hour": [
+        "Rain last 24h",
+        LENGTH_MILLIMETERS,
+        "mdi:weather-pouring",
+        None,
+    ],
     # new in json api (>1.0.0):
-    "rainlasthour": ["Rain last hour", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
-    "temperature_1d": ["Temperature 1d", TEMP_CELSIUS, "mdi:thermometer"],
-    "temperature_2d": ["Temperature 2d", TEMP_CELSIUS, "mdi:thermometer"],
-    "temperature_3d": ["Temperature 3d", TEMP_CELSIUS, "mdi:thermometer"],
-    "temperature_4d": ["Temperature 4d", TEMP_CELSIUS, "mdi:thermometer"],
-    "temperature_5d": ["Temperature 5d", TEMP_CELSIUS, "mdi:thermometer"],
-    "mintemp_1d": ["Minimum temperature 1d", TEMP_CELSIUS, "mdi:thermometer"],
-    "mintemp_2d": ["Minimum temperature 2d", TEMP_CELSIUS, "mdi:thermometer"],
-    "mintemp_3d": ["Minimum temperature 3d", TEMP_CELSIUS, "mdi:thermometer"],
-    "mintemp_4d": ["Minimum temperature 4d", TEMP_CELSIUS, "mdi:thermometer"],
-    "mintemp_5d": ["Minimum temperature 5d", TEMP_CELSIUS, "mdi:thermometer"],
-    "rain_1d": ["Rain 1d", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
-    "rain_2d": ["Rain 2d", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
-    "rain_3d": ["Rain 3d", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
-    "rain_4d": ["Rain 4d", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
-    "rain_5d": ["Rain 5d", LENGTH_MILLIMETERS, "mdi:weather-pouring"],
+    "rainlasthour": ["Rain last hour", LENGTH_MILLIMETERS, "mdi:weather-pouring", None],
+    "temperature_1d": [
+        "Temperature 1d",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        None,
+        DEVICE_CLASS_TEMPERATURE,
+    ],
+    "temperature_2d": [
```

Comment:
These are not current values but statistics. So shouldn't be device class temperature?

Reply:
This is ok now.